pandas/tests/indexing/test_at.py

```diff
@@ -105,12 +103,15 @@ def test_at_setitem_multiindex(self):
             np.zeros((3, 2), dtype="int64"),
             columns=MultiIndex.from_tuples([("a", 0), ("a", 1)]),
         )
-        df.at[0, "a"] = 10
+        df.at[0, ("a", 0)] = 10
+        df.at[0, ("a", 1)] = 10
         expected = DataFrame(
             [[10, 10], [0, 0], [0, 0]],
             columns=MultiIndex.from_tuples([("a", 0), ("a", 1)]),
         )
         tm.assert_frame_equal(df, expected)
+        with pytest.raises(TypeError, match=""):
```

Comment:
Personally, I would put this into a separate test. One test for the happy path and one for the failing path. YMMV

Reply:
I don't want to split this because this isn't actually a test I created, but one I had to modify based on the code change, so I'd like to keep the change as small as I reasonably can.

Reply:
Well, fair enough.

Personally I am really reluctant to modify existing tests. So I would rather add a new one. YMMV